### PR TITLE
Fix Issue #3075 by using None for local fields instead of Some(false)

### PR DESCRIPTION
**Pull request by Neshura87**
Link to Issue: https://github.com/LemmyNet/lemmy/issues/3075

### CODE REVIEW

File: crates/apub/src/protocol/objects/group.rs [hunk 1]
```diff
@@ -110,7 +110,7 @@ impl Group {
       deleted: Some(false),
       nsfw: Some(self.sensitive.unwrap_or(false)),
       actor_id: Some(self.id.into()),
-      local: Some(false),
+      local: None,
```
Comment (Nutomic) [inline crates/apub/src/protocol/objects/group.rs:113]: Actually this line shouldnt be changed, please revert it. Changing the line below is enough to fix the problem.

Comment (Neshura87) [reply crates/apub/src/protocol/objects/group.rs]: Changed in [87e31d3](https://github.com/LemmyNet/lemmy/pull/3088/commits/87e31d3979cf1f46b9dd37f1ce7071ad698508ee)

**Review by Nutomic — APPROVED** [review]

**Comment by Nutomic** [comment]
@dessalines `comment_view::tests::test_crud` `comment_view::tests::test_languages` keep failing randomly.

**Comment by snowe2010** [comment]
Did this fix #3198 ?

**Pull request merged**